docs/source/main_classes/pipelines.rst

```diff
@@ -30,15 +30,15 @@ Parent class: Pipeline
 .. autoclass:: transformers.Pipeline
     :members: predict, transform, save_pretrained
 
-NerPipeline
+TokenClassificationPipeline
 ==========================================
 
-.. autoclass:: transformers.NerPipeline
+.. autoclass:: transformers.TokenClassificationPipeline
 
 TokenClassificationPipeline
 ==========================================
 
-This class is an alias of the :class:`~transformers.NerPipeline` defined above. Please refer to that pipeline for
+This class is an alias of the :class:`~transformers.TokenClassificationPipeline` defined above. Please refer to that pipeline for
```

Comment:
I don't see the difference in names. Is it still an alias?

Reply:
Yes I think you meant to replace the title by `NerPipeline` @julien-c

Reply:
indeed, thanks